### PR TITLE
clonezilla documentation

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: docs/user_guide/7_restore_backup.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
+# Shadow Backup USB stick
+
+You can only follow these steps if you have been provided with a Shadow Backup USB stick. It has a 500 MB Clonezilla partition and a several GB for Clonezilla disk images. These can be used to restore any Shadow-provided laptop(s) and/or NUC(s) to their exact configuration (it's a full disk image restore) after the delivery was tested in Shadow's offices but before it was shipped to the customer. Doing this means losing all the data on the laptop(s) and/or NUC(s) since the original shipment of the equipment from Shadow.
+
```
Comment (biofotis) [inline docs/user_guide/7_restore_backup.md:8]: so please back up your data to an external device. I would make this bold to make sure they understand that.

File: docs/user_guide/7_restore_backup.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
+# Shadow Backup USB stick
+
+You can only follow these steps if you have been provided with a Shadow Backup USB stick. It has a 500 MB Clonezilla partition and a several GB for Clonezilla disk images. These can be used to restore any Shadow-provided laptop(s) and/or NUC(s) to their exact configuration (it's a full disk image restore) after the delivery was tested in Shadow's offices but before it was shipped to the customer. Doing this means losing all the data on the laptop(s) and/or NUC(s) since the original shipment of the equipment from Shadow.
+ 
+ The steps for restoring a NUC with Shadow Backup USB stick are the same as for laptop, except that the USB stick be connected to the NUC, as well as a monitor with HDMI connector (not provided) and a USB keyboard (not provided), and you have to choose a different image to restore from the relevant list in the Clonezilla process. Apart from that, the steps are the same.
+
```
Comment (biofotis) [inline docs/user_guide/7_restore_backup.md:12]: I think the first sentence needs some refactoring to make it clear they need keyboard and monitor.

File: docs/user_guide/7_restore_backup.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+ You don't have to restore both NUC and laptop. You can choose to restore just 1 of them.
+
+## Clonezilla backup restore steps (here device means either laptop or NUC)
+
+1. Connect the Shadow USB stick to the device you want to restore
+
+2. Power on the device while tapping F2 and F12 on the keyboard. Disable Secure Boot in BIOS for the device by following the instructions [here](https://www.gigabyte.com/us/Support/FAQ/3001) (the menus might look like slightly different to these screenshots)
```
Comment (biofotis) [inline docs/user_guide/7_restore_backup.md]: maybe mention here this step is already done by us? so its optional

File: docs/user_guide/7_restore_backup.md [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+
+3. Power on the device while tapping F7 and F8 on the keyboard. A blue dialog box will appear asking you which disk to boot from: select the Shadow backup USB 500 MB Clonezilla partition (it might show up as a SanDisk device, or a similar brand matching the model of the USB stick)
```
Comment (biofotis) [inline docs/user_guide/7_restore_backup.md]: Can you check if different F button needs to be pressed on laptop and nuc?

**Review by biofotis — CHANGES_REQUESTED** [review]

File: docs/user_guide/7_restore_backup.md (hunk 1 above)
Comment (ghost) [reply docs/user_guide/7_restore_backup.md:8]: Fixed in the last commit

File: docs/user_guide/7_restore_backup.md (hunk 2 above)
Comment (ghost) [reply docs/user_guide/7_restore_backup.md:12]: Fixed in the last commit

File: docs/user_guide/7_restore_backup.md (hunk 3 above)
Comment (ghost) [reply docs/user_guide/7_restore_backup.md]: Fixed in the last commit

File: docs/user_guide/7_restore_backup.md (hunk 4 above)
Comment (ghost) [reply docs/user_guide/7_restore_backup.md]: Fixed in the last commit

**Review by biofotis — APPROVED** [review]

**Review by beatrizleon — APPROVED** [review]

**Pull request merged**